### ISSUE TITLE
Magisk detection

### ISSUE THREAD

**Issue by devnoname120** [gauravssnl/BypassRootCheckPro#7]
![Screenshot_20240323-214440_SwissID.png](https://github.com/gauravssnl/BypassRootCheckPro/assets/2824100/941bbcb4-e6c0-4800-a76c-a6f5e6e5a91f)

https://play.google.com/store/apps/details?id=com.swisssign.swissid.mobile

**Comment by lssong99** [comment]
Did you change the Magisk Manager by MagiskHide？

**Comment by Richyblingz** [comment]
Yes i did  worked for other app execpt the overwallet app

Yahoo Mail: Search, Organize, Conquer 
 
  On Sat, Apr 27, 2024 at 4:09 AM, ***@***.***> wrote:   


Did you change the Magisk Manager by MagiskHide？

**Comment by devnoname120** [comment]
@lssong99 Does it work for you to start the app?

**Comment by Richyblingz** [comment]
Yes it works but the issue is that the wallet app closes and when i uncheck it from the bypass it open normal but flag the phone is rooted 

Yahoo Mail: Search, Organize, Conquer 
 
  On Sat, Apr 27, 2024 at 9:42 AM, ***@***.***> wrote:   


@lssong99 Does it work for you to start the app?

**Comment by devnoname120** [comment]
@Richyblingz I'm confused, what do you mean with the “wallet” app?

**Comment by Richyblingz** [comment]
What i mean is that my Overwallet app keeps closing after apply it to the BypassRootCheckPro

Yahoo Mail: Search, Organize, Conquer 
 
  On Sun, Apr 28, 2024 at 11:51 AM, ***@***.***> wrote:   


@Richyblingz I'm confused, what do you mean with the “wallet” app?

**Comment by devnoname120** [comment]
@Richyblingz Then it's off topic. We are talking about the Swiss ID app here, not Overwallet. I suggest you create a new issue for your problem.

**Comment by Richyblingz** [comment]
Ok thanks anyways

**Comment by lssong99** [comment]
@devnoname120 I don't have the Swiss ID app (nor could I install it, location restriction) Just by looking at the error message, want to confirm if you change the name of your Magisk Manager app to anything else using Magiskahide of Magisk Manager. (I believe you already did).

The other additional step could be using ZygiskNext instead of Magisk's built-in Zygisk which is known to be detectable easily. (No matter how you hide it.) Change to ZygiskNext could hide detection to those apps (I have couple bank app works only after change to ZygiskNext.) Use ZygiskNext_mod if you are using A15beta.

The other additional step could add is to prevent apps looking for common root related app with LSPosed module like "Hide My App List". However it is more complex and you should be able to find a lot of "how-to" on this additional topic.

Hope it works for you!

**Comment by devnoname120** [comment]
@lssong99 Thank you I will take a look and apply your suggestions. Note that I don't have a Swiss Play Store account either so I'm using this APK: https://apkpure.com/swissid/com.swisssign.swissid.mobile

Btw I confirm that it works with the latest Shamiko, but enabling Shamiko breaks some other apps that only work when the denylist is enforced.

**Edit**: I installed ZygiskNext and SwissID immediately stopped from detecting Magisk, even without enabling Shamiko or BypassRootCheckPro. 👌 It needs to be in denylist though.

**Comment by lssong99** [comment]
> @lssong99 Thank you I will take a look and apply your suggestions. Note that I don't have a Swiss Play Store account either so I'm using this APK: https://apkpure.com/swissid/com.swisssign.swissid.mobile
> 
> Btw I confirm that it works with the latest Shamiko, but enabling Shamiko breaks some other apps that only work when the denylist is enforced.
> 
> **Edit**: I installed ZygiskNext and SwissID immediately stopped from detecting Magisk, even without enabling Shamiko or BypassRootCheckPro. 👌 It needs to be in denylist though.

Glad to hear ZygiskNext works for you! Just be careful that Zygisk Next doesn't comptatible with A15 and due to some stupid issue (not author's fault) it was also stopped updating. You need ZygiskNext_mod for A15 system.

Maybe you could close this issue since it could be resolved by other means. (Maybe not BypassRootPro's scope to resolve.)

**Comment by devnoname120** [comment]
@lssong99 Btw, where can I find the ZygiskNext_mod that you mention?

**Comment by lssong99** [comment]
> @lssong99 Btw, where can I find the ZygiskNext_mod that you mention?

It's available on Telegram (yes... You need to step out the confort zone...) Search a channel '5ec1cff' (without quote) and you shall find it. latest version is 1.0.5 (305-d463cbb-release).

**Comment by devnoname120** [comment]
@lssong99 Ah that one is a mod? The file is named `Zygisk-Next-1.0.5-305-d463cbb-release.zip`.

**Comment by lssong99** [comment]
> @lssong99 Ah that one is a mod? The file is named `Zygisk-Next-1.0.5-305-d463cbb-release.zip`.

yes. It is.
